### PR TITLE
docs: add documentation about configuration

### PR DESCRIPTION
**Pull request by smoya**
**Description**

This PR adds mainly documentation around how the application can be configured.
It also does a touch on the project roadmap and how to build the app from source.

**Related issue(s)**
https://github.com/asyncapi/event-gateway/issues/25

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -57,3 +59,33 @@ It could even be an automated task whenever you update your specs.
...
+
+# Roadmap
+The idea is to keep iterating and support all the protocols AsyncAPI supports through [bindings](https://github.com/asyncapi/bindings).  
+However, we reduced the scope for the first versions, so we can give support to the most used protocols. 
+
+The list, which can be subject to change, is:
+
+- [Kafka](https://kafka.apache.org) - Under development.
+- [MQTT](https://mqtt.org)
+- [AMQP](https://www.amqp.org)
```
Comment (jonaslagoni) [inline README.md:71]: Are these two under development, or just next on the list? Is it in that specific order or? I don't think this little list clarifies much to the end-user 😅

File: README.md [hunk 2]
```diff
@@ -57,3 +59,33 @@ It could even be an automated task whenever you update your specs.
...
+
+### Install from pre-compiled binaries
+TBD
+
+### Install from source
+This project is built with [Go](https://golang.org/), and it uses [Go Modules](https://golang.org/ref/mod) for managing dependencies.  
+The Minimum required version of Go is set in [go.mod](go.mod) file.
+
+1. Clone this repository.
+2. Run `make build`. The binary will be placed at `bin/out/event-gateway`.
```
Comment (jonaslagoni) [inline README.md:88]: Now that I have it build, do I have to do anything specific to run it? 🤔

File: README.md [hunk 3]
```diff
@@ -57,3 +59,33 @@ It could even be an automated task whenever you update your specs.
...
+
+## Configuration
+Please refer to [/config](/config).
```
Comment (jonaslagoni) [inline README.md:91]: Should this not refer to `/docs/config`?

File: README.md (hunk 2 above)
Comment (smoya) [reply README.md:88]: No. it's a binary. I guess you will want to just run it.

File: README.md (hunk 3 above)
Comment (smoya) [reply README.md]: Fixed

File: README.md (hunk 1 above)
Comment (smoya) [reply README.md]: I removed those mentions and specified that we aim to give support **only** to Kafka protocol for the first version.

Does it look better?

Comment (jonaslagoni) [reply README.md]: 👍

File: docs/config/README.md [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+Configuration for Event-Gateway is done via environment variables as well.
+
+## Global
+| Environment variable        | Type    | Description                                                    | Default | Required | examples                                                                                                    |
+| --------------------------- | ------- | -------------------------------------------------------------- | ------- | -------- | ----------------------------------------------------------------------------------------------------------- |
+| EVENTGATEWAY_DEBUG          | boolean | Enable or disable debug logs                                   | `false` | No       | `true`, `false`                                                                                             |
+| EVENTGATEWAY_ASYNC_API_DOC  | string  | Path or URL to a valid AsyncAPI doc (v2.0.0 is only supported) | `false` | No       | `/var/opt/streetlights.yml`, `https://github.com/asyncapi/spec/blob/master/examples/streetlights-kafka.yml` |
```
Comment (jonaslagoni) [inline docs/config/README.md:10]: Not 2.1?

**Review by jonaslagoni — APPROVED** [review]
👍

File: docs/config/README.md (hunk 4 above)
Comment (magicmatatjahu) [reply docs/config/README.md:10]: GoLang parser only supports the 2.0.0.

Comment (jonaslagoni) [reply docs/config/README.md:10]: Ahh 😄

**Review by magicmatatjahu — APPROVED** [review]

**Pull request merged**